assets/timber.scss.liquid

```diff
@@ -1833,6 +1899,256 @@ label.error {
   }
 }
 
+/*============================================================================
+  #Mobile Nav
+==============================================================================*/
+.mobile-nav {
```

Comment:
i'd maybe add that as comments then, it isn't clear on reading the file what does what